src/Editor.ts

```diff
@@ -227,17 +227,37 @@ export default class Editor extends EventDispatcher {
     return this.set(new TextDocument(new Delta().insert(text), selection), source);
   }
 
-  getActive() {
-    const { selection } = this.doc;
-    let active = selection
-      ? selection[0] === selection[1]
-      ? { ...this.activeFormats, ...this.doc.getLineFormat(selection) }
-      : { ...this.doc.getFormats(selection) }
-      : {};
-    Object.values(this.modules).forEach(module => {
-      if (module.getActive) active = { ...active, ...module.getActive() };
-    });
-    return active;
+  fixupSelection(selection) {
+    const fixedSelection = selection;
+    if (fixedSelection) {
```

Comment:
Why add a new const that just refers to selection?

Reply:
The intent was to make it immutable, but I forgot, that's basically just a pointer. So I changed it to const fixedSelection = [selection[0], selection[1]] and making sure selection is an array of two.